### DISCUSSION TITLE
ChromeOS as Host Platform and Target Platform

### DISCUSSION THREAD

**Discussion by tautologias345** [godotengine/godot-proposals#5687] [Platforms]
ChromeOS is a very famous operating system. Google is its owner. I want to use ChromeOS to export game to ChromeOS.

**Comment by dsnopek** [comment]
AFAIK, games can be created for ChromeOS in two ways:

1. As Android apps: https://developer.android.com/topic/arc
2. As Progressive Web Apps (PWAs): https://support.google.com/chromebook/answer/9658361?hl=en&co=GENIE.Platform=Desktop

Godot already supports exporting games as Android apps and PWAs!

Is there something in particular that you find missing?
